package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "site-recorder",
-  "version": "0.1.2",
+  "version": "0.1.4",
```

Comment:
you don't need to update version, it will be updated automatically by TC: http://tc.dev.wixpress.com/viewType.html?buildTypeId=SiteRecorder_SiteRecorder_O

Reply:
OK 10x